wrapper/src/test/java/integration/util/AuroraTestUtility.java

```diff
@@ -1166,15 +1214,27 @@ public String hostToIP(String hostname) {
   }
 
   public String hostToIP(String hostname, boolean fail) {
-    try {
-      final InetAddress inet = InetAddress.getByName(hostname);
-      return inet.getHostAddress();
-    } catch (UnknownHostException e) {
-      if (fail) {
-        fail("The IP address of host " + hostname + " could not be determined");
+    int remainingTries = 5;
```

Comment:
I was seeing occasional test failures where the host could not be determined, presumably this is an intermittent problem and it doesn't happen so hopefully the retry here will get rid of the issue